src/sql/statements.cpp

```diff
@@ -343,15 +350,15 @@ TableRef::~TableRef() {
   delete join;
   delete alias;
 
-  if (list != nullptr) {
+  if (list) {
     for (TableRef* table : *list) {
       delete table;
     }
     delete list;
   }
 }
 
-bool TableRef::hasSchema() const { return schema != nullptr; }
+bool TableRef::hasSchema() const { return static_cast<bool>(schema); }
```

Comment:
For me, the old version is clearer. But I'm not sure what is generally preferred.

Reply:
To me, comparing a pointer `==`/`!=` `nullptr` doesn't seem right at all (not only in if clauses). Thus, the change. Maybe @Bouncner has an opinion on that

Reply:
To me, comparing a pointer to `nullptr` is more obvious than casting it to another type which does implicitly the same.

Reply:
Alright, I'm outvoted :D